build.sbt

```diff
@@ -4,6 +4,7 @@ version := "1.0.0-SNAPSHOT"
 scalaVersion := "2.11.7"
 val smackVersion = "4.1.3"
 val akkaVersion = "2.3.12"
+val awsVersion = "0.5.+"
```

Comment:
personally i prefer a fixed version rather than a range. makes finding errors much easier. and there's tools to automatically find newer versions of your depencies, e.g. https://github.com/rtimush/sbt-updates